test/unit/BPool.t.sol

```diff
@@ -1434,9 +1431,8 @@ contract BPool_Unit_JoinPool is BasePoolTest {
   }
 
   function _assumeHappyPath(JoinPool_FuzzScenario memory _fuzz) internal pure {
-    vm.assume(_fuzz.initPoolSupply >= INIT_POOL_SUPPLY);
-    vm.assume(_fuzz.poolAmountOut >= _fuzz.initPoolSupply);
-    vm.assume(_fuzz.poolAmountOut < type(uint256).max / BONE);
+    _fuzz.initPoolSupply = bound(_fuzz.initPoolSupply, INIT_POOL_SUPPLY, type(uint256).max / BONE);
```

Comment:
`/ BONE` is safe here, as `poolAmountOut` needs to be bigger than `initPoolSupply`, and lower than `type(uint256).max / BONE` (which means, `initPoolSupply` needs to be lesser than  `type(uint256).max / BONE`